javawrapper/device/samples/send-sample/Main.java

```diff
@@ -0,0 +1,100 @@
+package javaWrapper;
```

Comment:
Let's rename this sample to send-serialized-event. We already have a send-event up there.